pom.xml

```diff
@@ -76,8 +76,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.440.x</artifactId>
-        <version>3307.v2769886db_63b_</version>
+        <artifactId>bom-2.462.x</artifactId>
```

Comment:
note https://github.com/jenkinsci/bom/pull/3677

Reply:
Will bring the bom for `2.479.x`.